src/components/FormSelect/index.js

```diff
@@ -6,17 +6,18 @@ import { pick } from 'lodash'
 import { bemifyClassName } from 'utils/helpers'
 
 const FormSelect = ({ input, label, values, className, defaultValue, ...props }) => {
-  const selectProps = pick(props, [])
-
+  const selectProps = pick(props, [])  
```

Comment:
What does this do?

Reply:
nothing really, just a removed space, I guess because there was a debugger placed on those lines, and after debugging we get rid of it ^^
selectedProps was there already on the develop branch